docs/_Sidebar.md

```diff
@@ -0,0 +1,29 @@
+## Inductiva API
+
+## Start
+
+* [Install and upgrade](Install)
+* [API access tokens](API-access-tokens)
+* [Start simulating](Start-simulating)
+
+## Simulate
+
+* [Simulators](Simulators)
+    - [DualSPHysics](DualSPHysics)
+    - [SPlisHSPlasH](SPlisHSPlasH)
+    - [OpenFOAM](OpenFOAM)
+    - [SWASH](SWASH)
+    - [XBeach](XBeach)
+    - [Reef3D](Reef3D)
+    - [FDS](FDS)
+    - [GROMACS](GROMACS)
+* [Scenarios](Scenarios)
```

Comment:
In this release, we have no scenarios. In fact, the base package will probably not point to any scenario from this side bar.